### PR TITLE
Fix #43 - Design Login Screen UI

### LINKED ISSUE #43: Login Screen
**Summary:**

Design the login screen

**Expected behaviour:**

Match the UI given

**Observed behaviour:**

Nothing 

**Device and Android version:**

Don't add any libraries

**Screenshots:**

Please share if necessary only

### PR DESCRIPTION
**Pull request by nishank95**
### Description
Added Login Screen UI as per the designed Mockup for Login Screen

Fixes #43 

### Type of Change:
**Delete irrelevant options.**

- User Interface
Changed the UI for Login Screen Activity.
Added the following drawable files:
  1. Background for Login Screen
  2. ic_phone image for edit text drawable
  3. Background for proceed button

### Screenshot

BEFORE:
N/A

AFTER:
login_screen

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
N/A

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by nishank95** [comment]
1. I will make the changes in style of action bar
2. Because when the device rotates the form elements are not accessible and hence used scroll view.

**Comment by Rondaul** [comment]
@nishank95  Use the scrollview in lanscape layout of activity_login. You can create lanscape by clicking on the orientation in the editior. Keep the scroll view there

File: app/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -1,16 +1,151 @@
...
 
-    <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:tools="http://schemas.android.com/tools"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:context=".ui.login.LoginActivity">
 
-    </android.support.constraint.ConstraintLayout>
+        <ScrollView
```
Comment (harikautilya) [inline app/src/main/res/layout/activity_login.xml]: Why do we need scroll view ? it's a single page view.If u wanna push the screen up on keyboard opening use adjust pan in manifest file. 
https://stackoverflow.com/questions/17410499/difference-between-adjustresize-and-adjustpan-in-android

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -1,16 +1,151 @@
...
+            tools:context=".ui.login.LoginActivity"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/login_page_bg"
+            >
+            <android.support.constraint.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+           >
+
```
Comment (harikautilya) [inline app/src/main/res/layout/activity_login.xml]: also pull from master and use the ids supplied

File: app/src/main/res/layout/activity_login.xml [hunk 3]
```diff
@@ -1,16 +1,151 @@
...
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="80dp"
+                    android:fontFamily="@font/grandhotel_regular"
+                    android:text="@string/login_tv_app_title"
+                    android:textColor="@android:color/white"
+                    android:textSize="48sp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
```
Comment (harikautilya) [inline app/src/main/res/layout/activity_login.xml]: Use defined Id here as well

File: app/src/main/res/layout/activity_login.xml [hunk 4]
```diff
@@ -1,16 +1,151 @@
...
+                    android:drawableStart="@drawable/ic_phone"
+                    android:ems="10"
+                    android:hint="@string/login_et_phone_no_hint"
+                    android:inputType="phone|number"
+                    android:padding="15dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="0.503"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/tv_app_title" />
+
```
Comment (harikautilya) [inline app/src/main/res/layout/activity_login.xml]: Use defined Id here as well

File: app/src/main/res/layout/activity_login.xml [hunk 5]
```diff
@@ -1,16 +1,151 @@
...
+                        android:layout_marginEnd="10dp"
+                        android:layout_marginLeft="24dp"
+                        android:layout_marginRight="10dp"
+                        android:layout_marginStart="24dp"
+                        android:layout_weight="1"
+                        android:background="@android:color/white"
+                        app:layout_constraintEnd_toStartOf="@+id/tv_label_or"
+                        app:layout_constraintStart_toStartOf="parent"
+                        tools:layout_editor_absoluteY="354dp" />
+
```
Comment (harikautilya) [inline app/src/main/res/layout/activity_login.xml]: Use defined Id here as well

File: app/src/main/res/layout/activity_login.xml (hunk 1 above)
Comment (nishank95) [reply app/src/main/res/layout/activity_login.xml]: Scroll view is used if phone is rotated

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (nishank95) [reply app/src/main/res/layout/activity_login.xml]: can you specifiy which id because in master branch>activity_login.xml file i cant find any id's

**Comment by Rondaul** [comment]
@nishank95 please put the scrollview only in landscape mode. no need in the portrait mode

**Comment by nishank95** [comment]
@Rondaul I have added a land/activity_login.xml for landscape mode which contains the scroll view and removed scroll view from portrait mode.If it's alright please let me know then i will commit the changes

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (harikautilya) [reply app/src/main/res/layout/activity_login.xml]: Please pull from master for the ids

File: app/src/main/res/layout/activity_login.xml (hunk 1 above)
Comment (harikautilya) [reply app/src/main/res/layout/activity_login.xml]: Develop the application only in portrait mode

Comment (nishank95) [reply app/src/main/res/layout/activity_login.xml]: Changed the LoginActivity to portrait mode

**Comment by Aaklii** [comment]
Let me know when it needs to be merged

**Comment by Aaklii** [comment]
@Kautilyahari @geekanamika @Rondaul @vvkvivekl 
Review the file before  I merge them/

**Comment by Rondaul** [comment]
@Aaklii tell him to use the ids provided in `ids.xml`

**Comment by nishank95** [comment]
Hey @Rondaul I have used the id's from ids.xml file you can check activity_login.xml file.

**Pull request merged**